eagleproject/core/views.py

```diff
@@ -298,12 +309,18 @@ def active_stake_stats():
 def address(request, address):
     if address != address.lower():
         return redirect("address", address=address.lower())
+    data = _address_transfers(address)
+    return render(request, "address.html", data)
+
+def _address_transfers(address):
     long_address = address.replace("0x", "0x000000000000000000000000")
     latest_block_number = _latest_snapshot_block_number()
+    block_number = latest_block_number - 4
```

Comment:
4 block confirms? Might recommend making this a setting/const somewhere.

Reply:
What I was trying to avoid is the case where the listener hasn't picked up the transactions yet, but the user's balance has increased or decreased due to the transfer. In hindsight, we refresh the DB every minute, so I'm going to play this safer and do two minutes worth, or conservatively 120 / 10 = 12 blocks.